solidity/scripts/liquidity-rewards-starter.js

```diff
@@ -39,105 +41,72 @@ const mintAndApproveLPReward = async (
     const lpRewardsKEEPTBTC = await LPRewardsKEEPTBTC.deployed()
-
-    const KEEPETHWrappedTokenAddress = await lpRewardsKEEPETH.wrappedToken()
-    const KEEPETHWrappedTokenContract = await TestToken.at(
-      KEEPETHWrappedTokenAddress
-    )
-
-    const TBTCETHWrappedTokenAddress = await lpRewardsTBTCETH.wrappedToken()
-    const TBTCETHHWrappedTokenContract = await TestToken.at(
-      TBTCETHWrappedTokenAddress
-    )
-
-    const KEEPTBTCWrappedTokenAddress = await lpRewardsKEEPTBTC.wrappedToken()
-    const KEEPTBTCHWrappedTokenContract = await TestToken.at(
-      KEEPTBTCWrappedTokenAddress
-    )
-
-    const keepToken = await KeepToken.at(KeepTokenAddress)
-
+    const lpRewardsTBTCSaddle = await LPRewardsTBTCSaddle.deployed()
     const reward = web3.utils.toWei("1000000")
 
-    await initLPRewardContract(
-      lpRewardsKEEPETH,
-      keepToken,
-      lpRewardsOwner,
-      reward
-    )
-    await initLPRewardContract(
-      lpRewardsTBTCETH,
-      keepToken,
-      lpRewardsOwner,
-      reward
-    )
-    await initLPRewardContract(
-      lpRewardsKEEPTBTC,
-      keepToken,
-      lpRewardsOwner,
-      reward
-    )
-
-    const staker1 = accounts[8]
-    const staker1WrappedTokenBalance = web3.utils.toWei("300")
-
-    const staker2 = accounts[9]
-    const staker2WrappedTokenBalance = web3.utils.toWei("700")
-
-    await mintAndApproveLPReward(
-      KEEPETHWrappedTokenContract,
-      lpRewardsKEEPETH.address,
-      staker1,
-      staker1WrappedTokenBalance
-    )
-    await mintAndApproveLPReward(
-      KEEPETHWrappedTokenContract,
-      lpRewardsKEEPETH.address,
-      staker2,
-      staker2WrappedTokenBalance
-    )
-
-    await mintAndApproveLPReward(
-      TBTCETHHWrappedTokenContract,
-      lpRewardsTBTCETH.address,
-      staker1,
-      staker1WrappedTokenBalance
-    )
-    await mintAndApproveLPReward(
-      TBTCETHHWrappedTokenContract,
-      lpRewardsTBTCETH.address,
-      staker2,
-      staker2WrappedTokenBalance
-    )
-
-    await mintAndApproveLPReward(
-      KEEPTBTCHWrappedTokenContract,
-      lpRewardsKEEPTBTC.address,
-      staker1,
-      staker1WrappedTokenBalance
-    )
-    await mintAndApproveLPReward(
-      KEEPTBTCHWrappedTokenContract,
-      lpRewardsKEEPTBTC.address,
-      staker2,
-      staker2WrappedTokenBalance
-    )
-
-    await lpRewardsKEEPETH.stake(staker1WrappedTokenBalance, { from: staker1 })
-    await lpRewardsKEEPETH.stake(staker2WrappedTokenBalance, { from: staker2 })
-
-    await lpRewardsTBTCETH.stake(staker1WrappedTokenBalance, { from: staker1 })
-    await lpRewardsTBTCETH.stake(staker2WrappedTokenBalance, { from: staker2 })
-
-    await lpRewardsKEEPTBTC.stake(staker1WrappedTokenBalance, { from: staker1 })
-    await lpRewardsKEEPTBTC.stake(staker2WrappedTokenBalance, { from: staker2 })
+    const LP_REWARDS = {
+      KEEP_ETH: {
+        contract: lpRewardsKEEPETH,
+        wrappedTokenContract: null,
```

Comment:
Do we have to define this property? Wouldn't it be enough to do `LP_REWARDS[key].wrappedTokenContract = await getWrappedTokenContract(...`?